pkg/k8s/watchers/node.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020-2021 Authors of Cilium
+// Copyright 2020 Authors of Cilium
```

Comment:
This diff should be inversed

Reply:
I'll fix this, but these copyright strings differ quite a bit all over the code base. I'm not sure if anyone is keeping up to date with them?

Reply:
That's fine, we try to remember to update them when we modify those files. In this case though, this diff shouldn't exist